3rd/datasketches/common/CMakeLists.txt

```diff
@@ -29,17 +29,18 @@ target_include_directories(common
 
 target_compile_features(common INTERFACE cxx_std_11)
 
-target_sources(common
-  INTERFACE
-    ${CMAKE_CURRENT_SOURCE_DIR}/include/common_defs.hpp
-    ${CMAKE_CURRENT_SOURCE_DIR}/include/memory_operations.hpp
-    ${CMAKE_CURRENT_SOURCE_DIR}/include/MurmurHash3.h
-    ${CMAKE_CURRENT_SOURCE_DIR}/include/serde.hpp
-    ${CMAKE_CURRENT_SOURCE_DIR}/include/count_zeros.hpp
-    ${CMAKE_CURRENT_SOURCE_DIR}/include/inv_pow2_table.hpp
-    ${CMAKE_CURRENT_SOURCE_DIR}/include/binomial_bounds.hpp
-    ${CMAKE_CURRENT_SOURCE_DIR}/include/conditional_back_inserter.hpp
-    ${CMAKE_CURRENT_SOURCE_DIR}/include/conditional_forward.hpp
-    ${CMAKE_CURRENT_SOURCE_DIR}/include/ceiling_power_of_2.hpp
-)
+install(TARGETS common EXPORT ${PROJECT_NAME})
```

Comment:
Why this change?

Reply:
I've just copy and paste the folders. They Changed this Cmake . if you look the previous version 2.1.0 and the newer version 3.3.0

Reply:
Yeah ok, I wasn't doing a full copy - this CMake file is handcrafted and needs to stay the same, no install. Also, we don't need the tests

Reply:
ok. I will do that for all datasketches folders. I saw also that build fail for ARM due to UVW update (and libuv). I will rollback  that to previous version